lib/read/lib/Cardano/Wallet/Read/Block/SlotNo.hs

```diff
@@ -0,0 +1,66 @@
+{-# LANGUAGE FlexibleContexts #-}
+
+module Cardano.Wallet.Read.Block.SlotNo (getSlotNo) where
```

Comment:
```suggestion
module Cardano.Wallet.Read.Block.Slot
    ( Slot (..)
    , getSlot
    ) where

…
newtype Slot = Slot { unSlot :: Natural }

getSlot :: EraFunK Block Slot
```

Same reasoning as for `BlockNo`, plus the fact that [formal specification calls it `Slot`](https://github.com/cardano-foundation/fine-types/blob/c18585b91dd6acc4df44c1ccb0404d7e245cbbd8/lib/cardano-ledger-specs/Babbage/Address.fine#L10) instead of `SlotNo`. Spec says

```
Slot = ℕ; --^ absolute slot number
```

Who am I to question it? 🤷 😂  Joking aside, I believe that matching the specification to the letter is important, as it eliminates small, but nonzero sources of mental load — "equal" is better than "almost the same".

Reply:
See change my mind up there